.github/workflows/arduino-cli.yml

```diff
@@ -22,6 +22,23 @@ jobs:
         arduino-cli lib install OneButton
         sed -i '/recipe.output.tmp_file={build.project_name}.hex/d' "$HOME/.arduino15/packages/STM32/hardware/stm32/1.8.0/platform.txt"
         sed -i '/recipe.output.save_file={build.project_name}.{build.variant}.hex/d' "$HOME/.arduino15/packages/STM32/hardware/stm32/1.8.0/platform.txt"
+    - name: Unit Test
```

Comment:
I'm wondering: would that be possible to relocate this task in another workflow (i.e. another file in `.github/workflows/`? This would allow to:

1. be able to generate and upload binaries even if tests fails (I really don't want to let tests behind but this whole project is moving so fast...)
2. run tests in parallel of other workflows (produce bins, generate docs, lint code)

It is not obvious whether it depends on the previous task or not...

Reply:
Done. 
The pression unit test does not pass anymore after the merge of the fixed point PR.
The test check the equality between the output of the floating point function and the fixed point one. The results are
input | truth | output
------|------|---------
0|0 |0
100|0 |0
1000|144 |144
5000|983 |984
8000|2181| 2182
16000|4543| 4545
24000|7837| 7839
65535|17761| 17762

Do you think the test should test for near equality instead?

Reply:
I think we can keep strict equality for now.

Reply:
That means that the unit test will fail

Reply:
If I understand right:

1. You have one test that compares the old pressure acquisition function with the new one
2. This test fails if you compare with strict equality

Am I right?

Do you think the following strategy is a good idea?

1. We compare with near equality in this test
2. We add a second test to check few values without using the old function (like you did before, with actual and absolute "inputValues == expectedOutputValue") and use strict equality

Reply:
IMHO both strategies are fine.

Reply:
@dsferruzza You understood correctly. As you can see in the table, the difference is at most 2. We could consider that the fixed point function is good enough an generate a new set of truth value from it and use strict equality. As you suggested, we can also keep the current test with near equality.

Reply:
will do this tonigh

Reply:
Done